src/test/java/com/revature/services/JMeterServicesTest.java

```diff
@@ -0,0 +1,291 @@
+package com.revature.services;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.fail;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+
+import org.apache.jmeter.control.LoopController;
+import org.apache.jmeter.protocol.http.sampler.HTTPSampler;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import com.revature.docutest.TestUtil;
+import com.revature.templates.LoadTestConfig;
+
+import io.swagger.models.HttpMethod;
+import io.swagger.models.Operation;
+import io.swagger.models.Path;
+import io.swagger.models.Swagger;
+
+class JMeterServicesTest {
+
+    private JMeterServices jm;
+    private LoadTestConfig loadConfig = new LoadTestConfig();
+    private static final String JMeterPropPath = "src/test/resources/test.properties";
+    private static final String CSV_FILE_PATH = "./datafiles/user_0.csv";
+    public static final String DIRECTORY_PATH = "./datafiles";
+
+    @BeforeAll
+    static void setUpBeforeClass() throws Exception {
+    }
+
+    @AfterAll
+    static void tearDownAfterClass() throws Exception {
+    }
+
+    @BeforeEach
+    void setUp() throws Exception {
+        loadConfig.setLoops(1);
+        loadConfig.setRampUp(2);
+        loadConfig.setThreads(10);
+        loadConfig.setDuration(-1);
+        loadConfig.setTestPlanName("JMeterServicesTest");
+
+        jm = new JMeterServices();
+        TestUtil.initFields();
+
+        File directory = new File(DIRECTORY_PATH);
+        deleteFolder(directory);
+    }
+
+    @AfterEach
+    void tearDown() throws Exception {
+    }
+
+    @Test
+    void testLoadTestingLoop() {
+        loadConfig.setLoops(2);
+        jm.loadTesting(TestUtil.get, loadConfig, JMeterPropPath);
+
+        try (BufferedReader reader = new BufferedReader(new FileReader(CSV_FILE_PATH))) {
+            int counter = 0;
+            int expectedReq = (loadConfig.getLoops() * loadConfig.getThreads());
+
+            while (reader.readLine() != null) {
+                counter++;
+            }
+            counter--; // decrement for header line
+            System.out.println("Expected Request Count: " + expectedReq);
+            System.out.println("Actual Request Count: " + counter);
```

Comment:
Perhaps these could be moved to a logger?

Reply:
I was planning on implementing logging later in a separate branch. Since we might want this output for debugging, I'll add a Todo comment for the time being.